app/assets/javascripts/modules/help.js.coffee

```diff
@@ -0,0 +1,253 @@
+
+#= require models/data/preferences
+
+data = @edsc.models.data
+ui = @edsc.models.ui
+ns = @edsc.models.page
+
+@edsc.help = do ($=jQuery, config=@edsc.config, wait = @edsc.util.xhr.wait, page=@edsc.page, PreferencesModel = data.Preferences) ->
+
+
+  tourOptions =
+    tour_end:
+      title: 'Tour Ended'
+      element: '#main-toolbar h1 a'
+      placement: 'bottom'
+      content: 'Your tour has ended.  At any point you may restart it by visiting the home page and clicking
+                on the "Take a Tour" link.'
+
+  defaultTemplate = "<div class='popover tour'>
+                <div class='arrow'></div>
+                <h3 class='popover-title'></h3>
+                <div class='popover-content'></div>
+                <div class='popover-navigation'>
+                  <div class='btn-group'>
+                    <button class='button-small button-outline' data-role='prev'>« Prev</button>
+                    <button class='button-small' data-role='next'>Next <i class='fa fa-arrow-circle-right'></i></button>
+                  </div>
+                  <button class='button-small button-outline pull-left' data-role='end'>End Tour</button>
+                  <label class='pull-left' style='padding-top: 5px;padding-left: 5px;'>
+                    <input data-role='toggleHideTour' class='toggleHideTour' type='checkbox' />
+                    <small>Do not show again</small>
+                  </label>
+                </div>
+              </div>"
+
+  tour = [{
+      title: "Search"
+      content: '<p>Use Earthdata Search Client\'s natural language processing-enabled search tool to quickly narrow
+                down to relevant collections.  An example search phrase could be "Land Surface Temperature over
+                Texas last month".  Results will be displayed in the collection panel below.</p>
+                <p>If you would prefer to pick a temporal range from a calendar (<i class="fa fa-clock-o"></i>),
```

Comment:
Let's change this to:
`<p>If you would prefer to pick a temporal range from a calendar (<i class="fa fa-clock-o"></i>) or manually set spatial boundaries (<i class="fa fa-fw fa-crop"></i>), use the buttons to the right of the search box.</p>`

Since you can't actually see the shapefile upload from this initial screen I feel like mentioning it specifically will cause confusion.